ext/mixed/js/display.js

```diff
@@ -931,6 +931,14 @@ class Display {
         return (typeof key === 'string' ? (key.length === 1 ? key.toUpperCase() : key) : '');
     }
 
+    _getNoteContext() {
+        return {
+            document: {
+                title: document.title
```

Comment:
This is the document of the popup itself. Needs to query the title from root frame `Frontend`.

Reply:
Something like https://github.com/siikamiika/yomichan/commit/b23621d1cacf86021b562601a15022c44d8cbffe in https://github.com/FooSoft/yomichan/issues/398 using `apiForward` with the root popup ID from the broadcast done in https://github.com/FooSoft/yomichan/pull/417

Reply:
Also, ID is not needed if popup depth is checked before responding.

Reply:
This brings up an issue with the iframe popups though, the wrong `Frontend` might respond if it has a popup proxy with the same ID as the root page.

Reply:
One more comment, the current way to get document context from the parent frame seems to be through `window.frontendInitializationData` where the URL is passed through the `Popup` -> `DisplayFloat.prepare` (-> ... -> `PopupProxy`) chain. Not sure which way would be the best for this use case, but at least the document title seems more dynamic than the URL although both can be modified after the page has loaded.

Reply:
> This is the document of the popup itself. Needs to query the title from root frame `Frontend`.

I know I had forgotten something and that this felt too simple.

> Not sure which way would be the best for this use case, but at least the document title seems more dynamic than the URL although both can be modified after the page has loaded.

Yeah, it's probably better to get the title dynamically since it's not a time-critical operation. And I was also aware that the URL is possible to change, but chose not to address the issue at that time due to added complexity.

Reply:
> And I was also aware that the URL is possible to change, but chose not to address the issue at that time due to added complexity.

It seems that additional permissions would be needed to monitor the URL change properly anyway: https://developer.chrome.com/extensions/tabs#event-onUpdated

Polling would work, but it's good to avoid it as much as possible. For clipboard it was the only choice. Caching is also an option, but depending on cache miss handling it can have problems with outdated data or periodical high latency.

Reply:
Updated in 696897f6fa431c0c24332c5dd2a2c12a07923c0e.

At some point in the future, it may be better to just pass the document information as part of the call to `setContent`, but for now, I want to avoid cluttering the `context` value that is already passed to that function. A lot of that code will likely be touched when addressing #402 anyways.